kind_registry.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
```

Comment:
this i will remove
this was used to setup local k8s and docker registry using kind